core/consensus_test.go

```diff
@@ -38,7 +38,10 @@ func buildBasicPreprepareMessage(
 		Type: proto.MessageType_PREPREPARE,
 		Payload: &proto.Message_PreprepareData{
 			PreprepareData: &proto.PrePrepareMessage{
-				Proposal:     proposal,
+				Proposal: &proto.ProposedBlock{
+					EthereumBlock: ethereumBlock,
+					Round:         0, //TODO:
```

Comment:
Would this be correct (or maybe even we could specify additional `round` parameter and use it instead if makes sense)?

```suggestion
					Round:         view.Round,
```

Reply:
Fixed e35e015f1148476754801a39ed7634a87f23c026